swagger-spec/nodes/provider_detail.yaml

```diff
@@ -0,0 +1 @@
+# /nodes/{node_id}/files/providers/{provider}/
```

Comment:
Well, looks like that endpoint is 502-ing everywhere. Not ideal. Looking into fixing it now...

In the meantime, it should be documented essentially the same as providers_list.yaml since it's just the provider detail view. And it will have the same response (just with one folder/provider object instead of a list of folders/providers).